float32 laser_depth
float32 laser_x
float32 laser_y
